src/js/multiDomainFirstParties.js

```diff
@@ -1318,7 +1319,6 @@ var multiDomainFirstPartiesArray = [
     "springerprofessional.de",
   ],
   ["sprint.com", "sprintpcs.com", "nextel.com"],
-  ["squareup.com", "cash.app", "mkt.com", "squarecdn.com"],
```

Comment:
This line should be getting removed.

Reply:
oh boy!  Sorry Square!!!  Fixing that now